README.md

```diff
@@ -1,14 +1,21 @@
-# vehicle-commander-qt
+# message-simulator-qt
 
-The Vehicle Commander Qt demonstrates developer samples for building and testing in-vehicle military applications with the ArcGIS Runtime Qt SDK.  The Vehicle Commander Qt projects contains source code for creating sample in-vehicle, touchscreen, and support applications.
+Message Simulator provides UDP messaging for applications and system components in the ArcGIS for the Military solution, including:
 
-![Image of Vehicle Commander]( ScreenShot.jpg "vehicle-commander-qt")
+* [Vehicle Commander](https://github.com/Esri/vehicle-commander-java)
+* [Squad Leader](https://github.com/Esri/squad-leader-android)
+* Custom transports and adapters for ArcGIS GeoEvent Extension for Server ([solutions-geoevent-java](https://github.com/Esri/solutions-geoevent-java))
```

Comment:
Might be better to link to actual GeoMessage adapter resources (just to save some looking for):

* http://www.arcgis.com/home/item.html?id=cd3ce7a01fdc46a48aa199bfd091dd50
* https://github.com/Esri/solutions-geoevent-java/blob/master/solutions-geoevent/adapters/geomessage-adapter/README.md